app/src/main/java/com/berniac/vocalwarmup/music/Note.java

```diff
@@ -9,6 +9,21 @@
     private NoteRegister noteRegister;
     private NoteValue noteValue;
 
+    @Override
```

Comment:
All methods should be at least below the construnctor. In particular equals and hashCode could be in the bottom since they are not so intersting